crates/move-prover-lean-backend/src/lean_backend/bytecode_translator.rs

```diff
@@ -0,0 +1,3224 @@
+
+        // Translate the bytecode instruction.
+        match bytecode {
+            Call(_, dests, oper, srcs, aa) => {
+                use Operation::*;
+                match oper {
+                    FreezeRef => unreachable!(),
+                    UnpackRef | UnpackRefDeep | PackRef | PackRefDeep => {
+                        // No effect
+                    }
+                    OpaqueCallBegin(_, _, _) | OpaqueCallEnd(_, _, _) => {
+                        // These are just markers.  There is no generated code.
+                    }
+                    WriteBack(node, edge) => {}
+                    IsParent(node, edge) => {}
+                    BorrowLoc => {}
+                    ReadRef => {}
+                    WriteRef => {}
+                    Function(mid, fid, inst) => {
+                        let inst = &self.inst_slice(inst);
+                        let module_env = env.get_module(*mid);
+                        let callee_env = module_env.get_function(*fid);
+
+                        let mut args_str = srcs.iter().cloned().map(str_local).join(" ");
+                        let dest_vars: Vec<String> = dests
+                            .iter()
+                            .cloned()
+                            .map(str_local)
+                            // Add implict dest returns for &mut srcs:
+                            //  f(x) --> x := f(x)  if type(x) = &mut_
+                            .chain(
+                                srcs.iter()
+                                    .filter(|idx| self.get_local_type(**idx).is_mutable_reference())
+                                    .cloned()
+                                    .map(str_local),
+                            )
+                            .collect();
+                        let dest_str = if dest_vars.len() > 1 {
+                            format!("({})", dest_vars.join(", "))
+                        } else {
+                            dest_vars.join("")
+                        };
+
+                        // special casing for type reflection
+                        let mut processed = false;
+
+                        // TODO(mengxu): change it to a better address name instead of extlib
+                        if env.get_extlib_address() == *module_env.get_name().addr() {
+                            let qualified_name = format!(
+                                "{}::{}",
+                                module_env.get_name().name().display(env.symbol_pool()),
+                                callee_env.get_name().display(env.symbol_pool()),
+                            );
+                            if qualified_name == TYPE_NAME_MOVE {
+                                assert_eq!(inst.len(), 1);
+                                todo!();
+                            } else if qualified_name == TYPE_INFO_MOVE {
+                                assert_eq!(inst.len(), 1);
+                                todo!();
```

Comment:
ah, I see you are using `todo!`, let's just make sure we do that everywhere

Reply:
I'll swap away from todo to a wip macro to prevent it from panicking whenever one is tripped.

Reply:
Correction: I'll use wip!() whenever skipping is preferred behavior (like with some unimplemented bytecode instructions), and todo!() whenever skipping isn't possible.

Reply:
👍